internal/pkg/manifest/validate.go

```diff
@@ -132,14 +159,21 @@ func (w *WorkerServiceConfig) Validate() error {
 	}
 	for ind, taskDefOverride := range w.TaskDefOverrides {
 		if err = taskDefOverride.Validate(); err != nil {
-			return fmt.Errorf(`validate taskdef_overrides[%d]: %w`, ind, err)
+			return fmt.Errorf(`validate "taskdef_overrides[%d]": %w`, ind, err)
 		}
 	}
+	if err = validateNoCircularDependencies(validateDependenciesOpts{
```

Comment:
Question 🙋🏼‍♀️  Why `validateNoCircularDependencies` instead of `validateDependencies`, like in lb and backend service?